projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -307,7 +307,7 @@ class_exists( 'Jetpack' ) ||
 					),
 					'hasPaidPlan'                   => $publicize->has_paid_plan(),
```

Comment:
For some reason I didn't need to make the call to `jetpack_heartbeat` after removing the blog sticker, so the plan data got refreshed anyway. I thought maybe we were doing it here, when checking for the paid plan, but it doesn't look like it. We do [force a check here](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/class-jetpack-social.php#L226) but that should only run on the main admin page. That does have the side effect we're looking for though, of forcing a refresh when we load the admin page.

Reply:
After testing this again with just Jetpack enabled, then the plan information isn't being refreshed on editor load, which is as expected.